pkg/cmd/releasenotes/options.go

```diff
@@ -39,32 +39,38 @@ func NewGitGenerateOptions(streams genericclioptions.IOStreams) *GenerateOptions
 	}
 }
 
-func (o GenerateOptions) Validate() error {
+func (o *GenerateOptions) Validate() error {
```

Comment:
nit: validate should be read-only

Reply:
shouldn't all receivers be consistent?

Reply:
it doesn't implement any interface so no ambiguity can happen. maybe consistency is bettter.